docs/rfcs/RFC0003-Build-Tree-Normalization.md

```diff
@@ -367,10 +369,9 @@ gemmla/
     therock_artifact_gemm-api.toml
```

Comment:
@amcamd weren't you going to discuss this?   gemm-api I thought we rejected. Wouldn't rocblas and rocsolver be under linear algebra.   What is exposed to the user?  gemmla -> linear-algebra